streams/piping/general.any.js

```diff
@@ -39,7 +38,8 @@ promise_test(t => {
   const fakeRS = Object.create(ReadableStream.prototype);
   const ws = new WritableStream();
 
-  return methodRejects(t, ReadableStream.prototype, 'pipeTo', fakeRS, [ws]);
+  return promise_rejects_js(t, TypeError, ReadableStream.prototype.pipeTo.apply(fakeRS, [ws]),
+    'pipeTo should reject with a TypeError');
```

Comment:
...Okay, I lied. There was one helper (`methodRejects`) that was still being used. 😛 But it was only used in two places, and I found that inlining the helper made the tests more readable.